static/app/views/settings/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -242,54 +240,31 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
   renderTopButton(disabledFromFeatures: boolean, userHasAccess: boolean) {
```

Comment:
i'd try to replace this function with the new component. Each of these `render*` functions shuold likely be different components.